pkg/gardenlet/bootstrap/util/util.go

```diff
@@ -38,6 +43,42 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
+const (
+	// GardenerSeedBootstrapper is a constant for the gardener seed bootstrapper name.
+	GardenerSeedBootstrapper = "gardener.cloud:system:seed-bootstrapper"
+	// DefaultSeedName is the default seed name in case the gardenlet config.SeedConfig is not set
+	DefaultSeedName = "<ambiguous>"
+	// DedicatedSeedKubeconfig is a constant for the target cluster name when the gardenlet is using a dedicated seed kubeconfig
+	DedicatedSeedKubeconfig = "configured in .SeedClientConnection.Kubeconfig"
+	// InCluster is a constant for the target cluster name  when the gardenlet is running in a Kubernetes cluster
+	// and is using the mounted service account token of that cluster
+	InCluster = "in cluster"
```

Comment:
Hm, those constants look strange (why there are "sentences")? 

Edit: OK, I see they already existed before this PR, but still, somewhat strange.